ddtrace/opentracer/tracer.py

```diff
@@ -164,6 +159,27 @@ def start_span(self, operation_name=None, child_of=None, references=None,
             active `Scope` and creates a root `Span`.
         :return: an already-started Span instance.
         """
+        return self._start_span(
+            operation_name=operation_name,
+            child_of=child_of,
+            references=references,
+            tags=tags,
+            start_time=start_time,
+            ignore_active_span=ignore_active_span
+        )
+
+    def _start_span(self, operation_name=None, child_of=None, references=None,
```

Comment:
do we need this level of indirection? It's just another call in the stack without any immediate benefit (or at least I'm missing it)